src/PKSim.Core/Model/ParameterExtensions.cs

```diff
@@ -178,19 +181,19 @@ public static bool CanBeDefinedAsAdvanced(this IParameter parameter)
          return parameter.CanBeVariedInPopulation && !parameter.IsChangedByCreateIndividual;
       }
 
-      public static IReadOnlyList<IParameter> AllGlobalMoleculeParameters(this IEnumerable<IParameter> parameters)
+      public static IReadOnlyList<IParameter> AllGlobalMoleculeParameters(this IReadOnlyList<IParameter> parameters)
       {
-         return parameters.Where(x => x.NameIsOneOf(
-            CoreConstants.Parameters.REFERENCE_CONCENTRATION,
-            CoreConstants.Parameters.HALF_LIFE_LIVER,
-            CoreConstants.Parameters.HALF_LIFE_INTESTINE)
-         ).ToList();
+         return new[]
```

Comment:
The order in the simulation was never the one in individual because of this implementation. Changed!